src/org/usfirst/frc/team694/robot/commands/auton/SameSideSwitchAutonCommand.java

```diff
@@ -15,5 +16,9 @@ public SameSideSwitchAutonCommand() {
         addSequential(new DrivetrainRotateDegreesPIDCommand(-90)); //TODO: Make another magic number in FieldMap
```

Comment:
can you complete the todos?

Reply:
also get rid of the extra white space

Reply:
I'd rather have Pak do it since he is more familiar with field map.